OpenRA.Game/GameRules/WeaponInfo.cs

```diff
@@ -42,6 +42,8 @@ public sealed class WeaponInfo
 		[Desc("The sound played when the weapon is fired.")]
 		public readonly string[] Report = null;
 
+		public readonly bool OnlyPlayReportOnFirstBurst = false;
```

Comment:
In my opinion such yaml hacks are not much better, and with two weapons at once you'd have to use FireDelay for the silent weapon on top of that to match the sound's timing with the bursts.
Also, two weapons that actually deal damage aren't great if we ever decide to stray from the "true to original" approach with regard to balancing.

FWIW, I could *really* use this downstream as well.